transports/azure-service-bus/legacy/migration.md

```diff
@@ -32,6 +32,12 @@ Before migrating endpoints to the forwarding topology, all endpoints must have m
 
 After applying the above steps to one or more endpoints, they should be deployed to production. This cycle should be repeated until all endpoints using the endpoint-oriented topology have migration mode enabled.
 
+### Other Platform components
+
+If ServiceControl is configured in the [multi-instance mode](/servicecontrol/servicecontrol-instances/distributed-instances.md), it should be upgraded to Version 3.5.0 or later. The migration mode is enabled automatically. Upgrading Monitoring instances or single-instance ServiceControl is not required at this stage.
```

Comment:
Because a single instance SC does not subscribe external events and the migration mode is not required for publishers.

Reply:
True, but if a system is migrated and all subscribers are migrated, wouldn't it make sense to migrate publisher as well? Otherwise, when subscribers (endpoints) are moved to Forwarding topology, events published by SC that has not gone through migration will not be consumed. It's not a big deal, but it does mean some notifications will be lost. Thoughts @SzymonPobiega ?

Reply:
Would is solve the problem is we stated that the second step of the migration (moving from migration topology to forwarding proper) should be done by first migrating SC? Then we should not lose anything.

Or would it be easier to understand for customers to just say "always switch SC to migration topology"?